specification/columns/skumetername.md

```diff
@@ -0,0 +1,42 @@
+# SKU Meter Name
+
+The SKU Meter Name describes the capability being metered or measured by a particular SKU in a charge.
+
+Providers often have billing models in which multiple SKUs exist for a given service to describe and bill for different capabilities for that service. For example, an object storage service may have separate SKUs for capabilities such as object storage, API requests, data transfer, encryption, object management. This field helps practitioners understand which capabilities are being metered by the different SKUs that appear in a FOCUS dataset.
+
+The SkuMeterName column adheres to the following requirements:
+
+* SkuMeterName MUST be present in the billing data when when the provider includes a [SkuId](#skuid).
+* SkuMeterName MUST be of type String.
+* SkuMeterName MUST be null when SkuId is null.
+* SkuMeterName SHOULD NOT be null when SkuId is not null.
+
+## Examples
```

Comment:
I've never seen an examples section.  Are we adding this pattern?  Otherwise, I've seen examples as part of the top-level description

Reply:
We have an examples section in SkuPriceDetails!

I'll leave it in for now and we can remove later if needed.